src/indicator.js

```diff
@@ -112,14 +118,17 @@ export default class RunCatIndicator extends PanelMenuButton {
 
 	repaintUi() {
 		/** @type {CharacterState} */
-		const characterState = this.#data?.cpu > this.#settings.idleThreshold ? 'active' : 'idle'
+		const isActive = this.#data?.cpu > this.#settings.idleThreshold || this.#settings.reverseSpeed;
```

Comment:
I think that `idleThreshold` is not compatible with `invertRunningSpeed` setting. If utilization is 100%, it's getting harder to run for cat, but it would be weird if he goes to sleep :D WDYT?

If you agree with statement above, you could do something like:
```js
let utilization = this.#data?.cpu
let isActive = this.#data?.cpu > this.#settings.idleThreshold

if (this.#settings.invertRunningSpeed) {
	utilization = 100 - utilization
	isActive = true
}
```
And that's it. But you have to update settings UI, see the next comment

Reply:
I hope i'm not mistaken, but this `const isActive = this.#data?.cpu > this.#settings.idleThreshold || this.#settings.invertRunningSpeed` should ignore the `idleThreshold` setting if `invertRunningSpeed`  is active.

So the cat isn't sleeping, despite a idleThreshold value greater than 0.